### ISSUE TITLE
2.12.1 wants to change preferred-install option where order matters

### ISSUE THREAD

**Issue by Tobion** [ergebnis/composer-normalize#644]
I updated from 2.11.0 to 2.12.1 and now the normalization asks for this change

```diff
     "config": {
         "preferred-install": {
-            "acme/*": "source",
-            "*": "dist"
+            "*": "dist",
+            "acme/*": "source"
         },
```

This is wrong as the order matters in this config, see https://getcomposer.org/doc/06-config.md#preferred-install

**Comment by localheinz** [comment]
@Tobion 

Thank you for reporting!

**Comment by localheinz** [comment]
@Tobion 

Fix is under way!

**Comment by localheinz** [comment]
@Tobion 

Released as [`ergebnis/composer-normalize:2.12.2`](https://github.com/ergebnis/composer-normalize/releases/tag/2.12.2)!

Thank you again for reporting!

**Comment by Tobion** [comment]
Thanks for fixing so quickly!
